src/mappings.py

```diff
@@ -0,0 +1,157 @@
+        mapping_relation_element = external_reference.find("DisorderMappingRelation")
+        if mapping_relation_element is not None:
+            mapping_relation = mapping_relation_element.find("Name").text
+
+        validation_status_element = external_reference.find("DisorderMappingValidationStatus")
+        if validation_status_element is not None:
+            validation_status = validation_status_element.find("Name").text
+
+        # Check if DisorderMappingICDRelation and DisorderMappingICDRefUri elements exist
+        icd_relation_element = external_reference.find("DisorderMappingICDRelation")
+        if icd_relation_element is not None:
+            icd_relation = icd_relation_element.find("Name").text
+
+        icd_ref_uri_element = external_reference.find("DisorderMappingICDRefUri")
+        if icd_ref_uri_element is not None:
+            icd_ref_uri = icd_ref_uri_element.text
+
+        # Append values to lists
+        orpha_codes.append(orpha_code)
+        names.append(name)
+        sources.append(source)
+        references.append(reference)
+        mapping_relations.append(mapping_relation)
+        validation_statuses.append(validation_status)
+        icd_relations.append(icd_relation)
+        icd_ref_uris.append(icd_ref_uri)
+
+    # Create DataFrame
+    data = {
+        "OrphaCode": orpha_codes,
+        "Name": names,
+        "Source": sources,
+        "Reference": references,
+        "DisorderMappingRelation": mapping_relations,
+        "DisorderMappingValidationStatus": validation_statuses,
+        "DisorderMappingICDRelation": icd_relations,
+        "DisorderMappingICDRefUri": icd_ref_uris
+    }
+    df = pd.DataFrame(data)
+    return df
+
+def run(
+    input_nomenclature_xml: str = INPUT_NOMENCLATURE_XML, input_mms_matches_tsv: str = INPUT_MMS_MATCHES_TSV,
+    input_sssom_config: str = INPUT_CONFIG, outpath: str = OUTPUT_FILE
+):
+    """Run"""
+    # Get data
+    df: pd.DataFrame = xml_as_df(input_nomenclature_xml)
+
+    # Create / rename columns
+    df['predicate_id'] = df['DisorderMappingRelation'].apply(
+        lambda x: 'skos:exactMatch' if x.startswith('E (')
+        else 'skos:narrowMatch' if x.startswith('NTBT (')
+        else 'skos:broadMatch' if x.startswith('BTNT (')
+        else 'skos:relatedMatch')  # 0 instances of relatedMatch; just a fallback
+    df['subject_id'] = df['OrphaCode'].apply(lambda x: 'Orphanet:' + str(x))
+    df['object_id'] = df['DisorderMappingICDRefUri'].apply(lambda x: 'icd11.foundation:' + str(x) if x else None)
+    df = df.rename(columns={'Name': 'subject_label'})
+
+    # Filter by only if ICD11Foundation ref exists && is exact match
+    df = df[(df['DisorderMappingICDRefUri'].notna()) & (df['predicate_id'] == 'skos:exactMatch')]
+
+    # Get only columns we care about
+    df = df[['subject_id', 'subject_label', 'predicate_id', 'object_id']]
+    write_sssom(df, input_sssom_config, outpath)
```

Comment:
## Output
- [ ] Review @twhetzel @matentzn 

[ordo-icd11.sssom.tsv.zip](https://github.com/monarch-initiative/icd11/files/14809306/ordo-icd11.sssom.tsv.zip)


Example:
|subject_id|subject_label|predicate_id|object_id|
|-|-|-|-|
|Orphanet:58|Alexander disease|skos:exactMatch|icd11.foundation:2023359698|
|Orphanet:213|Cystinosis|skos:exactMatch|icd11.foundation:733715856|
|Orphanet:447|Paroxysmal nocturnal hemoglobinuria|skos:exactMatch|icd11.foundation:859588467|

Reply:
please make sure the mapping set is valid sssom using `sssom validate`, see

https://github.com/mapping-commons/mh_mapping_initiative/blob/e6154b97ef04becaf707ff89b9a79817c835ac4e/Makefile#L22

as an example.

I am assuming object_label is not available!

Reply:
- [ ] also add `mapping_provider` property with a link to the orphanet website)

Reply:
Done and done!
While working on the `sssom validate` bit, I noticed a problem, so I made an issue:
- https://github.com/mapping-commons/sssom-py/issues/512

Reply:
Can't believe I forgot to do this before, but here's this output as a google sheet: [ordo-icd11.sssom](https://docs.google.com/spreadsheets/d/1KZAeAyqYtjk7y-p7H7RUmBTT5rWX0KlBTdqlbX03Gc8/edit#gid=621796084)

`mondo-icd11.sssom` coming next